src/job/job.spec.js

```diff
@@ -49,8 +49,13 @@ describe("job", () => {
         const job = await xo.call("job.get", { id });
         expect(job).toMatchSnapshot({
           id: expect.any(String),
+          paramsVector: expect.any(Object),
           userId: expect.any(String),
         });
+        expect(job.paramsVector).toEqual({
+          items: defaultJob.paramsVector.items,
+          type: "crossProduct",
```

Comment:
Why not simply test with the defaultJob paramsVector?

Reply:
because it depends: sometimes this is `type: crossProduct` and sometimes (for `job.set`) the result is `type: cross product`.

Reply:
It's not the case here. You can simply use defaultJob paramsVector for the test.

Reply:
My bad, :woman_facepalming: . Now, I understand what you mean.